config/menus/main.cfg

```diff
@@ -1,30 +1,29 @@
 bind ESCAPE [if (clear_ui 1) [] [show_ui main]]    // it all starts here
 
-show_main_player_preview = 1
-setpersist show_main_player_preview 1
-setcomplete show_main_player_preview 1
+defbvar showmainplayerpreview 1 $idfbitpersist
```

Comment:
This is a variable, not an alias. I removed the underscores to match every other variable, again for consistency. If the style of variable names (either all variables or CubeScript-defined variables) is to change, I'll change this.

@MoonPadUSer are new variables supposed to use underscores?

Reply:
You're right in that all *user-facing* variables (and this one is definitely one) should be consistent. See my other comment. I think we can leave this as-is, but we really have to brainstorm on a variable naming strategy for the future.

The biggest problem with changing the existing variables' names is that we would likely have to add compat aliases to avoid breaking existing scripts.